styles/styles.css

```diff
@@ -5,7 +5,7 @@
 }
 
```

Comment:
- [ ] This requirements are about your `readme.md` file 👌. If you fix these you will be approved in the next review. The first thing is you guys forgot to add a screenshot of your **AWESOME!** project 😄. 
- [ ] The second one is since it is a team project also consider all of you in authors, and promise me not to forget it again, ok 😉🍀?

Reply:
I promise man, thanks 😉 .